app/AppKernel.php

```diff
@@ -70,6 +70,10 @@ public function registerBundles()
             new AppBundle\AppBundle(),
         ];
 
+        // Symfony 3.3 moved server:* commands to another bundle
```

Comment:
imho we should simply update https://github.com/symfony-cmf/cmf-sandbox/blob/04b0e9876efce273af4db1b367c5cb7359a4fdb2/composer.json#L21 to say ^3.3. the sandbox is not about supporting as many versions as possible.